tools/fastqc/test/main.nf

```diff
@@ -58,13 +58,8 @@ Channel
 --------------------------------------------------------------------------------------*/
 
 workflow {
-    params.modules['fastqc'].publish_dir = "fastqc_single"
-    params.modules['fastqc'].suffix = "_single"
-    fastqcSingle ( params.modules['fastqc'], ch_fastq_single_end )
-
-    params.modules['fastqc'].publish_dir = "fastqc_paired"
-    params.modules['fastqc'].suffix = "_paired"
-    fastqcPaired ( params.modules['fastqc'], ch_fastq_paired_end )
+    fastqcSingle ( params.modules['fastqc_se'], ch_fastq_single_end )
```

Comment:
shouldnt the process names also be snakecase?

Reply:
good point

Reply:
Haha! I was about to say the same 👍